schemas/dbt_project.json

```diff
@@ -332,6 +332,17 @@
                 },
                 "+tags": {
                     "$ref": "#/$defs/string_or_array_of_strings"
+                },
+                "+column_types": {
+                    "type": "object",
+                    "patternProperties": {
+                        "": {
+                            "type": "string"
+                        }
+                    }
+                },
+                "+full_refresh": {
+                    "type": "boolean"
```

Comment:
Can you change this to the boolean_or_jinja_string def?

Reply:
Great catch! I just added changes.